### ISSUE TITLE
Block heights of BTC, LTC do not increase

### ISSUE THREAD

**Issue by ianeinser** [openware/peatio#1528]
Hello,

I am using v1.9. I got a problem. This is the output of blockchain.yml

#<Faraday::Error: -- HTTP 500 Internal Server Error --

-- Request URL --
http://bitcoind:18332/

-- Request Method --
POST

-- Request headers --
{"Authorization":"SECRET","User-Agent":"Faraday v0.14.0","Accept":"application/json","Content-Type":"application/json"}

-- Request body --
{"jsonrpc":"1.0","method":"getblock","params":["000000000032a2e699b23f93faec9d1e7b8d46ee16bbe547611cbe83298173af",2]}

-- Response headers --
{"content-type":"application/json","date":"Tue, 14 Aug 2018 12:08:08 GMT","content-length":"92","connection":"close"}

-- Response body --
{"result":null,"error":{"code":-1,"message":"Block not available (pruned data)"},"id":null}

>

The output of eth blockchain is good and block height increments but those of btc, ltc do not. What is wrong with my configuration or is there anything else?

**Comment by ysv** [comment]
@ianeinser share your blockchain configurations

**Comment by ianeinser** [comment]
I just figured out the problem. I set the block height of both BTC and LTC too high in blockchain menu, when I added the blockchain. Thus the system waits until it reaches the heights that I put then it increments the value. One needs to check the actual block heights from the website before adding a new blockchain in v1.9, otherwise one may misunderstand that peatio doesn't synchronize with the coin servers
